### DISCUSSION TITLE
Energy Period Selection for Daily totals

### DISCUSSION THREAD

**Discussion by daimonkor** [slipx06/sunsynk-power-flow-card#417] [Ideas]
Is the idea add support Energy Period Selection to render daily energy statuses?

**Comment by slipx06** [comment]
Hi. Can you give me an example, not sure I understand the feature request.

**Comment by daimonkor** [comment]
Hi, I have the current card https://github.com/flixlix/energy-period-selector-plus (as sample). I think for non current date - show only daily data (like disabled or readonly, without an animation), other - online

**Comment by Roving-Ronin** [comment]
IMHO seems a bit weird, given this is a POWER flow card, to then have e ENERGY from other days upon it.

**Comment by daimonkor** [comment]
> IMHO seems a bit weird, given this is a POWER flow card, to then have e ENERGY from other days upon it.

POWER flow contains DAILY ENERGY data (but it is power flow card). My idea is only extend logic daily data with help date selection period. So offcourse for prev/next days card can show only daily counters, other - miss. Daily data are daily grid energy, daily load energy, daily charge battery energy, daily discharge battery energy, etc.

**Comment by slipx06** [comment]
Going to park this in the ideas discussion.
